tests/planner/test_grid_route.py

```
@@ -102,3 +102,83 @@ def test_grid_route_fails_safe_on_malformed_observation() -> None:
+def test_clearance_map_all_free_grid() -> None:
+    """A grid with no obstacles should leave all cells at 0 clearance."""
+    planner = GridRoutePlannerAdapter()
+    blocked = np.zeros((4, 4), dtype=bool)
+    cm = planner._compute_clearance_map(blocked)
+    assert np.all(cm == 0.0)
+
+
+def test_astar_with_clearance_prefers_corridor_centre() -> None:
+    """With clearance penalty, A* should route through the centre of a corridor.
+
+    Setup: 3-cell-wide horizontal corridor (rows 1–3 of a 5×10 grid, all free;
```

Ruff (0.15.9)

[warning] Docstring contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF002)